WalletWasabi/WabiSabi/Backend/Rounds/Arena.Partial.cs

```diff
@@ -167,7 +167,7 @@ public async Task<ConnectionConfirmationResponse> ConfirmConnectionAsync(Connect
 		{
 			var round = GetRound(request.RoundId);
 			var alice = GetAlice(request.AliceId, round);
-			Prison.Ban(alice.Coin.Outpoint, round.Id);
+			Prison.CheatingDetected(alice.Coin.Outpoint, request.RoundId);
```

Comment:
No. `FailedToConfirm`means that you didn't confirm (it could be that you have connectivity problems or you just left). Trying to cheat the coordinator is something different, it means you are providing invalid data, what is worse than not confirming.